lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2643,7 +2705,7 @@ std::optional<lldb::addr_t> SwiftLanguageRuntime::TrySkipVirtualParentProlog(
 }
 
 llvm::Expected<lldb::addr_t> GetTaskAddrFromThreadLocalStorage(Thread &thread) {
-#if SWIFT_THREADING_USE_DIRECT_TSD
+#if !SWIFT_THREADING_USE_RESERVED_TLS_KEYS
```

Comment:
Oh you flipped the condition later?

Reply:
The `swift::tis_get_key` function is available only on Darwin. Using `SWIFT_THREADING_USE_RESERVED_TLS_KEYS` is to ensure that `swift::tls_get_key` exists. If that compiler condition is false, then this function (`GetTaskAddrFromThreadLocalStorage`) will return an error. On Linux and Windows we'll need to figure out how to implement this, where `thread_local` variables are used instead of TSD.